Resources/doc/security.md

```diff
@@ -0,0 +1,20 @@
+# Security
+
+To completely disable some operations from your application, refers to [Disabling operations](operations.md#disabling-operations).
+
+This bundle does not have custom logic for addressing security concerns. If you wish to do so, you have to rely on the features provided by Symfony. For instance, if you wish to restrict the access of some endpoints, you can use [Access controls directives](http://symfony.com/doc/current/book/security.html#securing-url-patterns-access-control):
+
```

Comment:
I personally prefer that you only redirect to Sf security doc component in order to not confused anyone.